docs/template_plugin/src/template_plugin.cpp

```diff
@@ -152,14 +152,24 @@ InferenceEngine::QueryNetworkResult Plugin::QueryNetwork(const InferenceEngine::
     // So we need store as supported either unsupported node sets
     std::unordered_set<std::string> supported;
     std::unordered_set<std::string> unsupported;
-    auto opset = ngraph::get_opset4();
+    ngraph::OpSet op_super_set;
+#define _OPENVINO_OP_REG(NAME, NAMESPACE) op_super_set.insert<NAMESPACE::NAME>();
```

Comment:
Can we use `get_opsetN()`?
In other case we cannot use grep to find all places with opsets in order to add new opset.

Reply:
We should get all operations from all opsets :) The comment from @ilya-lavrenov was about previous technical decision. The details:
![image](https://user-images.githubusercontent.com/22008923/144017227-bf8f6d17-bd43-4050-bab9-fbde1d71737c.png)

Reply:
@ilyachur ?